go.mod

```diff
@@ -118,6 +118,7 @@ require (
 	github.com/shopspring/decimal v1.4.0 // indirect
 	github.com/sirupsen/logrus v1.9.3 // indirect
 	github.com/spf13/cast v1.7.1 // indirect
+	github.com/stretchr/objx v0.5.2 // indirect
```

Comment:
wondering why this is coming from, as I don't see it being used and there is no other dependency update.

Reply:
It is a dependency of testify's `mock` in [nodedport_test.go](https://github.com/utkuozdemir/pv-migrate/pull/336/files#diff-575b07b8ba7eecb94d7b6713cf43296439e4c7ae0f0ea2630a71379cfc8543c1R11)